.github/workflows/ci.yaml

```diff
@@ -77,3 +77,13 @@ jobs:
     - name: Upload code coverage report
       if: matrix.os == 'ubuntu-latest'
       uses: codecov/codecov-action@v4
+
+  baipp:
```

Comment:
I added one run of the baipp action into ci. This way the sdist and wheel will be checked during PRs and pushes to master. "baipp" is a suggestion.

The two jobs could also be called "test" and "build".